wfl/cli/cli_options.py

```diff
@@ -7,7 +7,7 @@ def _to_ConfigSet(ctx, param, value):
     return ConfigSet(value)
 
 def inputs(f):
-    """Create ConfigSet for given filename(s)"""
+    """Add a standard option for the OutputSpec outputs"""
```

Comment:
New docstrings for `inputs(f)` and `outputs(f)` are switched, I think.

Reply:
of course they are... Swapped back!